ci/cron/daily-compat.yml

```diff
@@ -196,3 +196,61 @@ jobs:
         env:
           GCRED: $(GOOGLE_APPLICATION_CREDENTIALS_CONTENT)
       - template: ../daily_tell_slack.yml
+
+  - job: blackduck_scan
+    timeoutInMinutes: 1200
+    pool:
+      name: linux-pool
+      demands: assignment -equals default
+    steps:
+      - checkout: self
+      - bash: ci/dev-env-install.sh
+        displayName: 'Build/Install the Developer Environment'
+      - bash: ci/configure-bazel.sh
+        displayName: 'Configure Bazel'
+        env:
+          IS_FORK: $(System.PullRequest.IsFork)
+          # to upload to the bazel cache
+          GOOGLE_APPLICATION_CREDENTIALS_CONTENT: $(GOOGLE_APPLICATION_CREDENTIALS_CONTENT)        
+      - template: ../bash-lib.yml
+        parameters:
+          var_name: bash_lib
+      - bash: ./build.sh "_$(uname)"
+        displayName: 'Build'
+      - bash: |
+          set -euo pipefail
+          eval "$(./dev-env/bin/dade-assist)"
+          bash <(curl -s https://raw.githubusercontent.com/DACH-NY/security-blackduck/master/synopsys-detect) \
+          ci-build digital-asset_daml master \
+          --logging.level.com.synopsys.integration=DEBUG \
+          --detect.tools=BAZEL \
+          --detect.bazel.target=//... \
+          --detect.bazel.dependency.type=haskell_cabal_library \
+          --detect.notices.report=true \
```

Comment:
Why do we need a notices report here? From the docs it sounds like this just creates a file and we never read that file or store it?

Reply:
setting this to true ensures that the haskell dependency processing completes before pressing on to scan of everything else in the part 2 scan